packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -275,7 +275,6 @@ const EAPostsItem = ({classes, ...props}: EAPostsListProps) => {
                 Wrapper={TitleWrapper}
                 read={isRead && !showReadCheckbox}
                 isLink={false}
-                wrap
```

Comment:
Was this just unnecessary then?

Reply:
Right, since we don't want titles to wrap in post items on desktop. (On mobile they seem to always wrap.)